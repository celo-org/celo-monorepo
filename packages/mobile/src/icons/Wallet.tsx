import colors from '@celo/react-components/styles/colors'
import * as React from 'react'
import Svg, { Rect } from 'svgs'

interface Props {
  height?: number
  color?: string
  width?: number
}
export default class Wallet extends React.Component<Props> {
  static defaultProps = {
    width: 27,
    height: 18,
    color: colors.dark,
  }

  render() {
    return (
      <Svg
        height={this.props.height}
        width={this.props.width}
        viewBox="0 0 31 22"
        fill="none"
        xmlns="http://www.w3.org/2000/svg"
      >
        <Rect
          x="1"
          y="1"
          width="27"
          height="20"
          rx="3"
          fill="#fff"
          stroke={this.props.color}
          strokeWidth="2"
        />
        <Rect
          x="20"
          y="8.5"
          width="10"
          height="5"
          rx="2"
          fill="#fff"
          stroke={this.props.color}
          strokeWidth="2"
        />
      </Svg>
    )
  }
}
