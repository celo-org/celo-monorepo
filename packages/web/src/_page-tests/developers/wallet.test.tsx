import * as React from 'react'
import * as renderer from 'react-test-renderer'
import Wallet from '../../../pages/developers/wallet'

describe('Wallet', () => {
  it('renders', () => {
    const tree = renderer.create(<Wallet />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
