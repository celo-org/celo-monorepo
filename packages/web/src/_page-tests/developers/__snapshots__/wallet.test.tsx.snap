// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallet renders 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "backgroundColor": "rgba(46,51,56,1.00)",
      "marginTop": "75px",
      "zIndex": 10,
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "zIndex": 20,
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitAlignSelf": "center",
          "WebkitBoxAlign": "center",
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "vertical",
          "WebkitBoxPack": "center",
          "WebkitFlexDirection": "column",
          "WebkitJustifyContent": "center",
          "alignItems": "center",
          "alignSelf": "center",
          "flexDirection": "column",
          "justifyContent": "center",
          "marginBottom": "30px",
          "marginTop": "30px",
          "maxWidth": "100vw",
          "msFlexAlign": "center",
          "msFlexDirection": "column",
          "msFlexItemAlign": "center",
          "msFlexPack": "center",
          "msGridRowAlign": "center",
          "paddingLeft": "10px",
          "paddingRight": "10px",
          "width": "100%",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitBoxDirection": "normal",
            "WebkitBoxFlex": 0,
            "WebkitBoxOrient": "horizontal",
            "WebkitBoxPack": "justify",
            "WebkitFlexDirection": "row",
            "WebkitFlexGrow": 0,
            "WebkitFlexShrink": 0,
            "WebkitJustifyContent": "space-between",
            "flexDirection": "row",
            "flexGrow": 0,
            "flexShrink": 0,
            "justifyContent": "space-between",
            "msFlexDirection": "row",
            "msFlexNegative": 0,
            "msFlexPack": "justify",
            "msFlexPositive": 0,
            "paddingBottom": "10px",
            "paddingLeft": "10px",
            "paddingRight": "10px",
            "paddingTop": "10px",
            "width": "100%",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitBoxFlex": 1,
              "WebkitBoxPack": "center",
              "WebkitFlexBasis": "0%",
              "WebkitFlexGrow": 1,
              "WebkitFlexShrink": 1,
              "WebkitJustifyContent": "center",
              "flexBasis": "0%",
              "flexGrow": 1,
              "flexShrink": 1,
              "justifyContent": "center",
              "marginLeft": "20px",
              "marginRight": "20px",
              "maxWidth": "350px",
              "msFlexNegative": 1,
              "msFlexPack": "center",
              "msFlexPositive": 1,
              "msFlexPreferredSize": "0%",
            }
          }
        >
          <svg
            fill="none"
            height="80"
            style={Object {}}
            viewBox="0 0 80 80"
            width="80"
          >
            <circle
              cx="40"
              cy="40"
              fill="#45CD85"
              r="40"
              style={Object {}}
            />
            <g
              style={
                Object {
                  "mixBlendMode": "screen",
                }
              }
            >
              <path
                clipRule="evenodd"
                d="M44.8334 49.8438C53.0314 49.8438 59.6771 43.198 59.6771 35C59.6771 26.802 53.0314 20.1562 44.8334 20.1562C36.6354 20.1562 29.9896 26.802 29.9896 35C29.9896 43.198 36.6354 49.8438 44.8334 49.8438ZM64.0521 35C64.0521 45.6142 55.4476 54.2188 44.8334 54.2188C34.2192 54.2188 25.6146 45.6142 25.6146 35C25.6146 24.3858 34.2192 15.7812 44.8334 15.7812C55.4476 15.7812 64.0521 24.3858 64.0521 35Z"
                fill="white"
                fillRule="evenodd"
              />
            </g>
            <g
              style={
                Object {
                  "mixBlendMode": "screen",
                }
              }
            >
              <path
                clipRule="evenodd"
                d="M35.4584 59.8438C43.6564 59.8438 50.3021 53.198 50.3021 45C50.3021 36.802 43.6564 30.1562 35.4584 30.1562C27.2604 30.1562 20.6146 36.802 20.6146 45C20.6146 53.198 27.2604 59.8438 35.4584 59.8438ZM54.6771 45C54.6771 55.6142 46.0726 64.2188 35.4584 64.2188C24.8442 64.2188 16.2396 55.6142 16.2396 45C16.2396 34.3858 24.8442 25.7812 35.4584 25.7812C46.0726 25.7812 54.6771 34.3858 54.6771 45Z"
                fill="white"
                fillRule="evenodd"
              />
            </g>
          </svg>
          <h1
            aria-level="1"
            className="css-reset-4rbku5 css-text-901oao"
            dir="auto"
            onClick={[Function]}
            role="heading"
            style={
              Object {
                "color": "rgba(255,255,255,1.00)",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "36px",
                "lineHeight": "40px",
                "marginTop": "20px",
                "textRendering": "geometricPrecision",
                "textTransform": "none",
              }
            }
          >
            Celo Test Wallet
          </h1>
          <h4
            aria-level="4"
            className="css-reset-4rbku5 css-text-901oao"
            dir="auto"
            onClick={[Function]}
            role="heading"
            style={
              Object {
                "color": "rgba(255,255,255,1.00)",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "24px",
                "lineHeight": "32px",
                "textRendering": "geometricPrecision",
                "textTransform": "none",
              }
            }
          >
            Temporarily Not Available
          </h4>
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitAlignItems": "center",
              "WebkitBoxAlign": "center",
              "WebkitBoxFlex": 1,
              "WebkitBoxPack": "center",
              "WebkitFlexBasis": "0%",
              "WebkitFlexGrow": 1,
              "WebkitFlexShrink": 1,
              "WebkitJustifyContent": "center",
              "alignItems": "center",
              "flexBasis": "0%",
              "flexGrow": 1,
              "flexShrink": 1,
              "height": "50vh",
              "justifyContent": "center",
              "maxWidth": "275px",
              "msFlexAlign": "center",
              "msFlexNegative": 1,
              "msFlexPack": "center",
              "msFlexPositive": 1,
              "msFlexPreferredSize": "0%",
            }
          }
        >
          <svg
            fill="#2E3338"
            height="100%"
            style={Object {}}
            viewBox="0 0 269 666"
            width="100%"
          >
            <path
              d="M242.076 664.653L19.3108 626.069C8.78103 624.25 1.12622 615.556 1.12622 605.426V51.1616C1.12622 40.6981 9.27489 31.82 20.2456 30.3848L242.993 1.01364C256.328 -0.738619 268.233 9.05734 268.233 21.7904V644.009C268.233 657.109 255.693 667.006 242.076 664.653Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M40.4972 129.199C40.5478 139.423 34.1538 148.185 26.2584 148.778C18.4434 149.371 12.1125 141.696 12.0628 131.657C12.0131 121.618 18.2626 112.84 26.0769 112.096C33.9552 111.318 40.4467 118.992 40.4972 129.199Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M83.2362 125.254C83.2881 135.752 76.6449 144.783 68.4667 145.413C60.3521 146.043 53.7999 138.156 53.7487 127.827C53.6976 117.497 60.1821 108.484 68.28 107.685C76.4573 106.886 83.1842 114.756 83.2362 125.254Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M129.577 121.845C129.629 132.355 122.509 141.386 113.743 142.038C105.059 142.673 98.0336 134.814 97.9825 124.485C97.9314 114.157 104.873 105.126 113.557 104.31C122.322 103.51 129.525 111.353 129.577 121.845Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M176.968 117.86C177.022 128.661 169.656 137.965 160.605 138.653C151.634 139.324 144.374 131.263 144.321 120.643C144.269 110.024 151.443 100.737 160.413 99.8847C169.479 99.0153 176.915 107.059 176.968 117.86Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M227.517 113.313C227.573 124.697 219.733 134.528 210.088 135.253C200.541 135.978 192.819 127.495 192.764 116.314C192.709 105.134 200.34 95.3214 209.886 94.3932C219.513 93.4646 227.461 101.929 227.517 113.313Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M40.5069 225.016C40.5562 234.961 34.1593 243.142 26.2619 243.328C18.4448 243.498 12.1133 235.706 12.0649 225.924C12.0164 216.142 18.2688 207.945 26.0852 207.629C33.9658 207.296 40.4578 215.087 40.5069 225.016Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M83.7235 223.722C83.7756 234.244 77.1306 242.925 68.9503 243.117C60.8334 243.292 54.2794 235.049 54.2282 224.695C54.1769 214.342 60.6631 205.662 68.7632 205.318C76.9427 204.957 83.6714 213.2 83.7235 223.722Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M130.072 221.891C130.125 232.716 123.005 241.646 114.237 241.842C105.55 242.038 98.5223 233.568 98.4697 222.928C98.417 212.288 105.359 203.359 114.045 202.994C122.812 202.612 130.018 211.065 130.072 221.891Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M177.477 220.601C177.53 231.424 170.162 240.35 161.11 240.56C152.136 240.753 144.874 232.296 144.821 221.654C144.768 211.013 151.945 202.088 160.917 201.713C169.985 201.321 177.423 209.777 177.477 220.601Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M228.039 218.766C228.094 229.882 220.25 239.068 210.603 239.265C201.053 239.477 193.33 230.781 193.276 219.864C193.222 208.947 200.857 199.778 210.406 199.367C220.036 198.972 227.984 207.65 228.039 218.766Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M40.5217 320.82C40.5724 331.059 34.175 339.143 26.2755 338.897C18.4563 338.649 12.122 330.292 12.0722 320.221C12.0223 310.151 18.2751 302.051 26.0936 302.146C33.9762 302.225 40.4711 310.599 40.5217 320.82Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M84.2082 321.664C84.2589 331.895 77.611 339.99 69.4284 339.735C61.3095 339.48 54.7549 331.121 54.705 321.055C54.6552 310.989 61.1443 302.894 69.2466 302.985C77.4282 303.06 84.1575 311.418 84.2082 321.664Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M177.985 323.344C178.039 334.168 170.669 342.698 161.614 342.429C152.638 342.143 145.374 333.305 145.321 322.663C145.268 312.021 152.447 303.475 161.422 303.58C170.492 303.667 177.932 312.521 177.985 323.344Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M228.563 324.686C228.618 335.807 220.772 344.567 211.123 344.284C201.57 343.984 193.846 334.903 193.792 323.981C193.738 313.059 201.374 304.283 210.925 304.385C220.558 304.47 228.508 313.566 228.563 324.686Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M40.5339 416.109C40.5845 426.33 34.1853 434.048 26.2837 433.383C18.4624 432.716 12.1266 424.037 12.0768 413.984C12.0271 403.931 18.2817 396.196 26.1022 396.695C33.9869 397.21 40.4834 405.905 40.5339 416.109Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M84.6931 419.627C84.7451 430.13 78.0964 438.071 69.9117 437.369C61.7905 436.668 55.2331 427.731 55.1819 417.396C55.1308 407.061 61.6205 399.104 69.725 399.637C77.9089 400.17 84.6411 409.123 84.6931 419.627Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M131.065 422.577C131.117 433.091 123.992 441.007 115.219 440.294C106.528 439.58 99.4976 430.631 99.4464 420.298C99.3953 409.965 106.342 402.017 115.033 402.566C123.804 403.098 131.013 412.063 131.065 422.577Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M178.494 426.085C178.547 436.887 171.175 445.021 162.118 444.258C153.14 443.51 145.874 434.294 145.821 423.674C145.768 413.053 152.949 404.92 161.926 405.486C170.999 406.051 178.44 415.283 178.494 426.085Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M229.085 430.139C229.142 441.529 221.294 450.076 211.642 449.259C202.087 448.442 194.36 438.718 194.304 427.532C194.249 416.346 201.886 407.766 211.44 408.397C221.075 409.01 229.029 418.75 229.085 430.139Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M40.5434 511.862C40.5923 521.756 34.1903 528.904 26.2868 527.838C18.4636 526.788 12.1274 518.063 12.0793 508.331C12.0311 498.599 18.2886 491.436 26.111 492.323C33.9977 493.226 40.4945 501.984 40.5434 511.862Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M85.1754 517.053C85.2271 527.514 78.5766 535.069 70.3896 533.918C62.2663 532.782 55.7072 523.512 55.6562 513.22C55.6053 502.927 62.0969 495.355 70.2035 496.323C78.3897 497.306 85.1236 506.592 85.1754 517.053Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M131.563 523.174C131.615 533.647 124.488 541.173 115.713 539.987C107.019 538.817 99.9872 529.542 99.9362 519.249C99.8853 508.956 106.834 501.415 115.527 502.405C124.301 503.41 131.511 512.717 131.563 523.174Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M179.002 528.754C179.055 539.481 171.681 547.165 162.621 545.933C153.641 544.716 146.373 535.188 146.321 524.641C146.268 514.094 153.451 506.394 162.43 507.431C171.505 508.467 178.949 518.027 179.002 528.754Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <path
              d="M229.607 534.518C229.661 545.565 221.811 553.445 212.157 552.145C202.599 550.861 194.871 541.019 194.817 530.169C194.763 519.319 202.404 511.407 211.96 512.494C221.598 513.597 229.552 523.471 229.607 534.518Z"
              stroke="#81868B"
              strokeMiterlimit="10"
            />
            <g
              style={
                Object {
                  "mixBlendMode": "screen",
                }
              }
            >
              <path
                className="r-animationKeyframes-kl28h1"
                d="M130.57 322.53C130.624 333.352 123.502 341.889 114.731 341.611C106.042 341.332 99.012 332.492 98.9594 321.857C98.9067 311.222 105.85 302.669 114.539 302.761C123.308 302.87 130.517 311.726 130.57 322.53Z"
                strokeMiterlimit="10"
                style={
                  Object {
                    "WebkitAnimationDuration": "5s",
                    "WebkitAnimationIterationCount": "infinite",
                    "WebkitAnimationTimingFunction": "ease-in-out",
                    "animationDuration": "5s",
                    "animationIterationCount": "infinite",
                    "animationTimingFunction": "ease-in-out",
                  }
                }
              />
            </g>
          </svg>
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-view-1dbjc4n"
    style={
      Object {
        "WebkitAlignSelf": "center",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "alignSelf": "center",
        "flexDirection": "column",
        "marginBottom": "50px",
        "marginTop": "50px",
        "maxWidth": "100vw",
        "msFlexDirection": "column",
        "msFlexItemAlign": "center",
        "msGridRowAlign": "center",
        "paddingLeft": "10px",
        "paddingRight": "10px",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "flex-start",
          "WebkitBoxAlign": "start",
          "WebkitBoxDirection": "normal",
          "WebkitBoxFlex": 0,
          "WebkitBoxLines": "multiple",
          "WebkitBoxOrient": "horizontal",
          "WebkitBoxPack": "justify",
          "WebkitFlexDirection": "row",
          "WebkitFlexGrow": 0,
          "WebkitFlexShrink": 0,
          "WebkitFlexWrap": "wrap",
          "WebkitJustifyContent": "space-between",
          "alignItems": "flex-start",
          "flexDirection": "row",
          "flexGrow": 0,
          "flexShrink": 0,
          "flexWrap": "wrap",
          "justifyContent": "space-between",
          "maxWidth": "100%",
          "msFlexAlign": "start",
          "msFlexDirection": "row",
          "msFlexNegative": 0,
          "msFlexPack": "justify",
          "msFlexPositive": 0,
          "msFlexWrap": "wrap",
          "paddingBottom": "10px",
          "paddingLeft": "10px",
          "paddingRight": "10px",
          "paddingTop": "10px",
          "width": "100%",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitBoxFlex": 1,
            "WebkitBoxPack": "start",
            "WebkitFlexBasis": "0%",
            "WebkitFlexGrow": 1,
            "WebkitFlexShrink": 1,
            "WebkitJustifyContent": "flex-start",
            "flexBasis": "0%",
            "flexGrow": 1,
            "flexShrink": 1,
            "justifyContent": "flex-start",
            "marginLeft": "20px",
            "marginRight": "20px",
            "marginTop": "30px",
            "maxWidth": "270px",
            "minWidth": "160px",
            "msFlexNegative": 1,
            "msFlexPack": "start",
            "msFlexPositive": 1,
            "msFlexPreferredSize": "0%",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitFlexBasis": "auto",
                "flexBasis": "auto",
                "height": "80px",
                "msFlexPreferredSize": "auto",
                "overflowX": "hidden",
                "overflowY": "hidden",
                "width": "80px",
                "zIndex": 0,
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "backgroundColor": "rgba(0,0,0,0.00)",
                  "backgroundPosition": "center",
                  "backgroundRepeat": "no-repeat",
                  "backgroundSize": "contain",
                  "bottom": "0px",
                  "height": "100%",
                  "left": "0px",
                  "position": "absolute",
                  "right": "0px",
                  "top": "0px",
                  "width": "100%",
                  "zIndex": -1,
                }
              }
            />
          </div>
          <h3
            aria-level="3"
            className="css-reset-4rbku5 css-text-901oao"
            dir="auto"
            onClick={[Function]}
            role="heading"
            style={
              Object {
                "color": "rgba(255,255,255,1.00)",
                "fontFamily": "Jost, futura-pt, futura, sans-serif",
                "fontSize": "24px",
                "lineHeight": "32px",
                "marginBottom": "20px",
                "marginTop": "20px",
                "textRendering": "geometricPrecision",
                "textTransform": "none",
              }
            }
          >
            Testnet Faucet
          </h3>
          <div
            className="css-text-901oao"
            dir="auto"
            onClick={[Function]}
            style={
              Object {
                "color": "rgba(221,221,221,1.00)",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            If you already have an account, faucet yourself to get more Celo Dollars.
          </div>
        </div>
        <div
          className="css-view-1dbjc4n"
          onClick={[Function]}
          onMouseDown={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitBoxDirection": "normal",
                "WebkitBoxFlex": 0,
                "WebkitBoxOrient": "horizontal",
                "WebkitBoxPack": "center",
                "WebkitFlexDirection": "row",
                "WebkitFlexGrow": 0,
                "WebkitJustifyContent": "center",
                "backgroundColor": "rgba(0,0,0,0.00)",
                "borderBottomColor": "rgba(0,0,0,0.00)",
                "borderLeftColor": "rgba(0,0,0,0.00)",
                "borderRightColor": "rgba(0,0,0,0.00)",
                "borderTopColor": "rgba(0,0,0,0.00)",
                "flexDirection": "row",
                "flexGrow": 0,
                "justifyContent": "center",
                "msFlexDirection": "row",
                "msFlexPack": "center",
                "msFlexPositive": 0,
                "opacity": 1,
                "outlineColor": "transparent",
                "width": "fit-content",
              }
            }
          >
            <a
              className="css-reset-4rbku5 css-cursor-18t94o4 css-text-901oao"
              data-focusable={true}
              dir="auto"
              href="/developers/faucet"
              onClick={[Function]}
              role="link"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "alignItems": "center",
                  "color": "rgba(53,208,127,1.00)",
                  "cursor": "pointer",
                  "display": "inline-flex",
                  "fontFamily": "Jost, futura-pt, futura, sans-serif",
                  "fontSize": "16px",
                  "fontWeight": "500",
                  "lineHeight": "16px",
                  "msFlexAlign": "center",
                  "textAlign": "center",
                  "textRendering": "geometricPrecision",
                }
              }
            >
              Get Testnet cUSD
              <div
                className="css-view-1dbjc4n"
                style={
                  Object {
                    "display": "inline-flex",
                    "paddingLeft": "0.4em",
                    "paddingTop": "0.185em",
                  }
                }
              >
                <svg
                  fill="none"
                  height="0.75em"
                  style={Object {}}
                  transform="rotate(0)"
                  viewBox="0 0 8 12"
                  width="0.75em"
                >
                  <path
                    clipRule="evenodd"
                    d="M1.19997 0L1.13249e-05 1.19996L4.80011 6.00006L0 10.8002L1.19996 12.0001L7.19977 6.00032L7.19953 6.00007L7.19979 5.99981L1.19997 0Z"
                    fill="#35D07F"
                    fillRule="evenodd"
                    opacity={1}
                    style={Object {}}
                  />
                </svg>
              </div>
            </a>
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitBoxFlex": 1,
            "WebkitBoxPack": "justify",
            "WebkitFlexBasis": "0%",
            "WebkitFlexGrow": 1,
            "WebkitFlexShrink": 1,
            "WebkitJustifyContent": "space-between",
            "flexBasis": "0%",
            "flexGrow": 1,
            "flexShrink": 1,
            "justifyContent": "space-between",
            "marginLeft": "20px",
            "marginRight": "20px",
            "marginTop": "30px",
            "maxWidth": "270px",
            "minWidth": "160px",
            "msFlexNegative": 1,
            "msFlexPack": "justify",
            "msFlexPositive": 1,
            "msFlexPreferredSize": "0%",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitFlexBasis": "auto",
                "flexBasis": "auto",
                "height": "80px",
                "msFlexPreferredSize": "auto",
                "overflowX": "hidden",
                "overflowY": "hidden",
                "width": "80px",
                "zIndex": 0,
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "backgroundColor": "rgba(0,0,0,0.00)",
                  "backgroundPosition": "center",
                  "backgroundRepeat": "no-repeat",
                  "backgroundSize": "contain",
                  "bottom": "0px",
                  "height": "100%",
                  "left": "0px",
                  "position": "absolute",
                  "right": "0px",
                  "top": "0px",
                  "width": "100%",
                  "zIndex": -1,
                }
              }
            />
          </div>
          <h3
            aria-level="3"
            className="css-reset-4rbku5 css-text-901oao"
            dir="auto"
            onClick={[Function]}
            role="heading"
            style={
              Object {
                "color": "rgba(255,255,255,1.00)",
                "fontFamily": "Jost, futura-pt, futura, sans-serif",
                "fontSize": "24px",
                "lineHeight": "32px",
                "marginBottom": "20px",
                "marginTop": "20px",
                "textRendering": "geometricPrecision",
                "textTransform": "none",
              }
            }
          >
            Send and Request Payments
          </h3>
          <div
            className="css-text-901oao"
            dir="auto"
            onClick={[Function]}
            style={
              Object {
                "color": "rgba(221,221,221,1.00)",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            Account start with 10 cUSD of Testnet funds. Try sending and requesting payments. Use the faucet page if you run out.
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitBoxFlex": 1,
            "WebkitBoxPack": "justify",
            "WebkitFlexBasis": "0%",
            "WebkitFlexGrow": 1,
            "WebkitFlexShrink": 1,
            "WebkitJustifyContent": "space-between",
            "flexBasis": "0%",
            "flexGrow": 1,
            "flexShrink": 1,
            "justifyContent": "space-between",
            "marginLeft": "20px",
            "marginRight": "20px",
            "marginTop": "30px",
            "maxWidth": "270px",
            "minWidth": "160px",
            "msFlexNegative": 1,
            "msFlexPack": "justify",
            "msFlexPositive": 1,
            "msFlexPreferredSize": "0%",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitFlexBasis": "auto",
                "flexBasis": "auto",
                "height": "80px",
                "msFlexPreferredSize": "auto",
                "overflowX": "hidden",
                "overflowY": "hidden",
                "width": "80px",
                "zIndex": 0,
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "backgroundColor": "rgba(0,0,0,0.00)",
                  "backgroundPosition": "center",
                  "backgroundRepeat": "no-repeat",
                  "backgroundSize": "contain",
                  "bottom": "0px",
                  "height": "100%",
                  "left": "0px",
                  "position": "absolute",
                  "right": "0px",
                  "top": "0px",
                  "width": "100%",
                  "zIndex": -1,
                }
              }
            />
          </div>
          <h3
            aria-level="3"
            className="css-reset-4rbku5 css-text-901oao"
            dir="auto"
            onClick={[Function]}
            role="heading"
            style={
              Object {
                "color": "rgba(255,255,255,1.00)",
                "fontFamily": "Jost, futura-pt, futura, sans-serif",
                "fontSize": "24px",
                "lineHeight": "32px",
                "marginBottom": "20px",
                "marginTop": "20px",
                "textRendering": "geometricPrecision",
                "textTransform": "none",
              }
            }
          >
            Exchange cUSD and CELO
          </h3>
          <div
            className="css-text-901oao"
            dir="auto"
            onClick={[Function]}
            style={
              Object {
                "color": "rgba(221,221,221,1.00)",
                "fontFamily": "EB Garamond, eb-garamond, Garamond, serif",
                "fontSize": "20px",
                "lineHeight": "28px",
                "marginBottom": "20px",
                "textRendering": "geometricPrecision",
              }
            }
          >
            Check the exchange rate and swap between Celo’s stable coin (cUSD) and variable value token (CELO) as you see fit.
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
